java/component/keyple-plugin/pcsc/src/main/java/org/eclipse/keyple/plugin/pcsc/PcscReader.java

```diff
@@ -14,21 +14,4 @@
 import org.eclipse.keyple.core.seproxy.event.ObservableReader;
 
 /** The PcscReader interface provides the public elements used to manage a PC/SC reader. */
-public interface PcscReader extends ObservableReader {
-  String SETTING_KEY_TRANSMISSION_MODE = "transmission_mode";
```

Comment:
I'm not sure there is an interest to move these parameters to PcscReaderSetting

Reply:
Not to PcscReaderSetting, but to PcscReaderConstants